src/views/virtualmachinesinstances/list/VirtualMachinesInstancesList.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import {
+  K8sResourceCommon,
+  ListPageBody,
+  ListPageCreate,
+  ListPageFilter,
+  ListPageHeader,
+  useK8sWatchResource,
+  useListPageFilter,
+  VirtualizedTable,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+import useVirtualMachinesInstancesColumns from '../hooks/useVirtualMachinesInstancesColumns';
+import { filters } from '../utils';
+
+import VirtualMAchinesInstancesRow from './VirtualMachinesInstancesRow';
+
+type VirtualMachinesInstancesListProps = {
+  kind: string;
+};
+
+const VirtualMachinesInstancesList: React.FC<VirtualMachinesInstancesListProps> = ({ kind }) => {
+  const { t } = useTranslation('plugin__kubevirt-plugin');
+
+  const [vmis, loaded, loadError] = useK8sWatchResource<V1VirtualMachineInstance[]>({
+    kind,
+    isList: true,
+    namespaced: true,
+  });
+  const [data, filteredData, onFilterChange] = useListPageFilter(vmis, filters);
+  const columns = useVirtualMachinesInstancesColumns();
+
+  return (
+    <>
+      <ListPageHeader title={t('Virtual Machines Instances')}>
+        <ListPageCreate groupVersionKind={kind}>
```

Comment:
Currently, im using the generic button of the component as Create VMI us is only yaml editor which is fine, if we will need any wizard/extender, it will move out, I'm trying to keep it as small as possible... :)

Reply:
I don't think we need a create btn